hnn_core/lfp.py

```diff
@@ -0,0 +1,234 @@
+"""
+LFPsim - Simulation scripts to compute Local Field Potentials (LFP) from cable
+compartmental models of neurons and networks implemented in NEURON simulation
+environment.
+
+LFPsim works reliably on biophysically detailed multi-compartmental neurons
+with ion channels in some or all compartments.
+
+Last updated 12-March-2016
+Developed by :
+Harilal Parasuram & Shyam Diwakar
+Computational Neuroscience & Neurophysiology Lab, School of Biotechnology,
+Amrita University, India.
+
+Email: harilalp@am.amrita.edu; shyam@amrita.edu
+www.amrita.edu/compneuro
+
+translated to Python and modified to use use_fast_imem by Sam Neymotin
+based on mhines code
```

Comment:
@cjayb Perhaps you should add a sentence here referencing the improvements/fixes you've implemented.

Reply:
Actually this is a bit inconsistent with how we document changes in this repository. I am almost thinking we should nuke this with a sentence or link to the old HNN repository from where we copied the code.

Reply:
Didn't see this, much nuked because the implementation is very far from anything in LFPsim! See my next round of commits...